### PR TITLE
Data member error message improvement

### PR DESCRIPTION
**Pull request by JindrichSusen**

### CODE REVIEW

File: backend/Origam.Schema.DeploymentModel.UI/Origam.Schema.DeploymentModel.UI.csproj [hunk 1]
```diff
@@ -97,6 +97,10 @@
       <Project>{976F3ABA-938D-4242-A93B-7D7A1861A66E}</Project>
       <Name>Origam.Schema</Name>
     </ProjectReference>
+    <ProjectReference Include="..\Origam.UI.Common\Origam.UI.Common.csproj">
```
Comment (tvavrda) [inline backend/Origam.Schema.DeploymentModel.UI/Origam.Schema.DeploymentModel.UI.csproj:100]: Why was this necessary?

Comment (JindrichSusen) [reply backend/Origam.Schema.DeploymentModel.UI/Origam.Schema.DeploymentModel.UI.csproj:100]: Yes. The project did not want to compile without it

**Review by tvavrda — APPROVED** [review]
